.github/workflows/test.yml

```diff
@@ -73,8 +73,10 @@ jobs:
           sudo apt remove bpfcc-tools
           zsh --version
 
-      - name: Run tests against all shells on alpine
+      - name: Run zsh tests with proper terminal setup
         run: |
+          export PS1="$ "
```

Comment:
Let's do these in `internal/test_helpers/zsh/your_shell.sh`, so that `make test_zsh` works directly without any additions

Reply:
May not work since we run sudo make_test which runs a different zsh but I'll try and see